mmv1/products/vertexai/IndexEndpointDeployedIndex.yaml

```diff
@@ -0,0 +1,224 @@
+    ignore_read: true
+    properties:
+      - !ruby/object:Api::Type::Integer
+        name: 'minReplicaCount'
+        description: The minimum number of replicas this DeployedModel will be always deployed on.
+        default_value: 2
+      - !ruby/object:Api::Type::Integer
+        name: 'maxReplicaCount'
+        description: The maximum number of replicas this DeployedModel may be deployed on when the traffic against it increases.
+        custom_expand: templates/terraform/custom_expand/vertex_ai_endpoint_deployed_index.go.erb
+  - !ruby/object:Api::Type::NestedObject
+    name: dedicatedResources
+    description: A description of resources that are dedicated to the DeployedIndex, and that need a higher degree of manual configuration.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'machineSpec'
+        description: The minimum number of replicas this DeployedModel will be always deployed on.
+        immutable: true
+        required: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'machineType'
+            description: The type of the machine.
+            immutable: true
+      - !ruby/object:Api::Type::Integer
+        name: 'minReplicaCount'
+        description: The minimum number of machine replicas this DeployedModel will be always deployed on. This value must be greater than or equal to 1.
+        immutable: true
+        required: true
+      - !ruby/object:Api::Type::Integer
+        name: 'maxReplicaCount'
+        description: The maximum number of replicas this DeployedModel may be deployed on when the traffic against it increases.
+        immutable: true
+  - !ruby/object:Api::Type::Boolean
+    name: enableAccessLogging
+    description: If true, private endpoint's access logs are sent to Cloud Logging.
+    default_value: false
+  - !ruby/object:Api::Type::NestedObject
+    name: deploymentIndexAuthConfig
+    description: If set, the authentication is enabled for the private endpoint.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: authProvider
+        description: Defines the authentication provider that the DeployedIndex uses.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: audiences
+            output: true
+            description: The list of JWT audiences. that are allowed to access. A JWT containing any of these audiences will be accepted.
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::Array
+            name: allowedIssuers
+            output: true
+            description: |
+              A list of allowed JWT issuers. Each entry must be a valid Google service account, in the following format: service-account-name@project-id.iam.gserviceaccount.com
+            item_type: Api::Type::String
```

Comment:
Are these both definitely outputs? [The docs](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.indexEndpoints#AuthProvider) don't indicate that's the case, and I believe providing these values is how the user controls access.